src/app/util/attribute-storage.cpp

```diff
@@ -291,6 +292,31 @@ CHIP_ERROR emberAfSetDynamicEndpoint(uint16_t index, EndpointId id, const EmberA
         }
     }
 
+    for (uint8_t i = 0; ep && (i < ep->clusterCount); i++)
+    {
+        if (!ep->cluster)
+        {
+            continue;
+        }
+
+        const EmberAfCluster * cluster = &(ep->cluster[i]);
+        if (!cluster->attributes)
+        {
+            continue;
+        }
+
+        for (uint16_t j = 0; j < cluster->attributeCount; j++)
+        {
+            const EmberAfAttributeMetadata * attr = &(cluster->attributes[j]);
+            if (emberAfAttributeSize(attr) > chip::app::Compatibility::Internal::gEmberAttributeIOBufferSpan.size())
+            {
+                ChipLogError(DataManagement,
+                             "Attribute %u (id=" ChipLogFormatMEI ") of Cluster %u (id=" ChipLogFormatMEI ") too large", j,
+                             ChipLogValueMEI(attr->attributeId), i, ChipLogValueMEI(cluster->clusterId));
```

Comment:
I'm not sure `i` and `j` are worth logging here; nothing even guarantees they will be stable.  The IDs should be the thing we log.

And instead of `too large` I would suggest logging both the size it is, and the max size allowed.

Reply:
Ack, will change.